wrangler.toml

```diff
@@ -1,4 +1,2 @@
 name = "unifi-cloudflare-ddns"
 main = "src/index.js"
-compatibility_date = "2023-04-23"
```

Comment:
Let's keep the `compatibility_date` property in the `wrangler.toml`

Reply:
Ah that's my bad, I thought this was used for node compat. Reverted :)